packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1687,8 +1688,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 summaryLogger,
             });
 
+            const summary = summarizeResult.summary.tree;
+            const summaryTree = summary[channelsTreeName] as ISummaryTree;
+            let all = 0;
+            let unreferenced = 0;
+            if (summaryTree.tree !== undefined) {
+                all = Object.keys(summaryTree.tree).length;
+                for(const key of Object.keys(summaryTree.tree)) {
+                    const dataStore = summaryTree.tree[key] as ISummaryTree;
+                    if (dataStore.unreferenced === true) {
+                        unreferenced ++;
+                    }
+                }
+            }
+
+            const topLevelSummaryStat: ITopLevelSummaryStats = {
+                dataStoreCount: all,
+                unreferencedDataStoreCount: unreferenced,
+                blobNodeCount: summarizeResult.stats.blobNodeCount,
+                treeNodeCount: summarizeResult.stats.treeNodeCount,
+                handleNodeCount: summarizeResult.stats.handleNodeCount,
+                totalBlobSize: summarizeResult.stats.totalBlobSize,
+            };
```

Comment:
```suggestion
            const topLevelSummaryStat: ITopLevelSummaryStats = {
                ...summarizeResult.stats,
                dataStoreCount: all,
                unreferencedDataStoreCount: unreferenced,
            };
```
Since `ITopLevelSummaryStats` extends `ISummaryStats`, just use the rest operator.